smart_card_connector_app/src/manifest.json.template

```diff
@@ -16,6 +16,13 @@ ${if PACKAGING=app}
     }
 ${endif}
   },
+${if PACKAGING=extension}
+  "browser_action": {
+    "default_icon": "icon.png",
+    "default_title": "Smart Card Connector",
```

Comment:
This is a hardcoded text in English, but ideally we should target i18n from day one.

Would `__MSG_appShortName__` work here? At least it should, according to this doc: https://developer.chrome.com/docs/extensions/reference/browserAction/#tooltip